example/aes.dart

```diff
@@ -0,0 +1,48 @@
+library example.aes;
+
+import "dart:convert";
+import "dart:typed_data";
+
+import "package:hex/hex.dart";
```

Comment:
No need, use standard library (included in `dart:convert`).

Reply:
I can't seem to find hex in here: https://api.dartlang.org/stable/2.0.0/dart-convert/dart-convert-library.html